code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1210,13 +1210,37 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	switch(H.nutrition)
 		if(NUTRITION_LEVEL_FULL to INFINITY)
 			H.throw_alert("nutrition", /obj/screen/alert/fat)
+			var/obj/item/organ/O = H.getorganslot(ORGAN_SLOT_HEART)
+			O.applyOrganDamage(0.25)
 		if(NUTRITION_LEVEL_HUNGRY to NUTRITION_LEVEL_FULL)
 			H.clear_alert("nutrition")
+			for(var/slot in list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_LUNGS, ORGAN_SLOT_APPENDIX, \
+	ORGAN_SLOT_EYES, ORGAN_SLOT_EARS, ORGAN_SLOT_TONGUE, ORGAN_SLOT_LIVER))
+				var/obj/item/organ/O = H.getorganslot(slot)
+				O.applyOrganDamage(-0.25)
 		if(NUTRITION_LEVEL_STARVING to NUTRITION_LEVEL_HUNGRY)
 			H.throw_alert("nutrition", /obj/screen/alert/hungry)
 		if(0 to NUTRITION_LEVEL_STARVING)
 			H.throw_alert("nutrition", /obj/screen/alert/starving)
-
+			for(var/slot in list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_LUNGS, ORGAN_SLOT_APPENDIX, \
+	ORGAN_SLOT_EYES, ORGAN_SLOT_EARS, ORGAN_SLOT_TONGUE, ORGAN_SLOT_LIVER))
+				var/obj/item/organ/O = H.getorganslot(slot)
+				O.applyOrganDamage(0.25)
+
+
+/datum/species/proc/set_click_cooldown(mob/living/carbon/human/H, num)
+	var/click_cooldown = world.time + ((num+H.next_move_adjust)*H.next_move_modifier)
+	if(!HAS_TRAIT(H, TRAIT_NOHUNGER))
```

Comment:
what the fuck is this shit right here

being slow is already a big problem in combat, why are you kicking people while they're down with CLICK COOLDOWNS

this also affects plasmaman and ethereal balance, because they can't get the 0.2 second click cooldown reduction that races that can eat can

how does this interact with gloves of the north star?

Reply:
click cooldowns > brute damage.

Reply:
get over it?